### DISCUSSION TITLE
No action audio module

### DISCUSSION THREAD

**Discussion by wsalmeida** [polybar/polybar#2994] [Q&A]
Hello the audio module is not having any effect it is configured left button when clicked mute the device and scroll increase the volume both do not work if anyone can help, What happens is that polybar always reads the hadset as main being the external speaker has no effect even if I switch to another device the polybar remains only recognized and does not apply the volume functions help please

**Comment by archusXIV** [comment]
Hi, which audio server do you use, alsa, pulseaudio, pipewire?
Post your module so we can guess...
